src/responder/ifp/org.freedesktop.sssd.infopipe.service.in

```diff
@@ -1,4 +1,5 @@
 [D-BUS Service]
 Name=org.freedesktop.sssd.infopipe
-Exec=@libexecdir@/sssd/sss_signal
+Exec=@ifp_exec@
```

Comment:
I think there is a typo; I cannot see  ifp_exec anywhere.
Sorry, I haven't finished review on Saturday